website/docs/cdktf/create-and-deploy/environment-variables.mdx

```diff
@@ -7,48 +7,17 @@ description: >-
 
 # Environment Variables
 
-Use these environment variables to configure CDK for Terraform behavior.
+Use these environment variables to configure CDK for Terraform (CDKTF) behavior.
+
+| Variable Name | Description | Default |
+| ------------- | ----------- | -------|
+| CDKTF_DISABLE_PLUGIN_CACHE_ENV | If set to `true` or `1`, CDKTF disables the [plugin cache](/cli/config/config-file#provider-plugin-cache). Otherwise, CDKTF creates the plugin cache directory in `~/.terraform.d/plugin-cache` if it is not already present. |  |
+| CDKTF_HOME | Where CDKTF should store cache and configuration files. | `~/.cdktf` |
+| CDKTF_LOG_FILE_DIRECTORY | The directory path where CDKTF should create `cdktf.log` and print logs at the `debug` level. If not set, CDKTF writes logs to standard out at the level specified in `CDKTF_LOG_LEVEL`. | standard out |
+| CDKTF_LOG_LEVEL | Controls how much log information CDKTF prints to the console. You can set it to one of the following values: `all`, `debug`, `info`, `warn`, `error`, `fatal`, `off`. | `debug` |
```

Comment:
I made an assumption that the default is `debug` PLEASE correct me if I'm wrong!

Reply:
The default is `warn`

Reply:
fixed!